libs/npm/core/src/components/grid/Grid.tsx

```diff
@@ -93,6 +97,9 @@ export const Grid = <T,>({
           })}
         </div>
         <div className="grid-table">
+          <Show visible={isLoading}>
```

Comment:
Have you tested this to determine if changing the `isLoading` property in the parent component makes this component re-render.  React state doesn't detect property changes.

Reply:
I tested in the local environment and it is showing the loading indicator on the editor content page.

Reply:
On the initial load it may.  What if you change the value from false to true again?  Depending on various scenarios properties do not cause a render when they change.

Reply:
Seems to be working when changing the filters as well.